esphome/components/pcd8544/pcd_8544.h

```diff
@@ -29,9 +29,11 @@ class PCD8544 : public PollingComponent,
   const uint8_t PCD8544_SETTEMP = 0x04;
   const uint8_t PCD8544_SETBIAS = 0x10;
   const uint8_t PCD8544_SETVOP = 0x80;
+  uint8_t contrast_ = 0x7f;
```

Comment:
Along with the changes above, you wont need to set the default value in cpp as the codegen will always set it. We don't want the default value in 2 different places.
```suggestion
  uint8_t contrast_;
```